src/transformers/models/wav2vec2_conformer/configuration_wav2vec2_conformer.py

```diff
@@ -74,6 +74,8 @@ class Wav2Vec2ConformerConfig(PretrainedConfig):
             The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
         layer_norm_eps (`float`, *optional*, defaults to 1e-12):
             The epsilon used by the layer normalization layers.
+        layer_norm_first (`bool`, *optional*, defaults to `True`):
```

Comment:
```suggestion
        post_layer_norm (`bool`, *optional*, defaults to `True`):
                Apply layer-norm after the Transformer layer. Otherwise, apply it in the feature projection and prior to the Transformer layer.
```